test/test_requirements.txt

```diff
@@ -0,0 +1,4 @@
+pytest
+TensorFlow 
+numpy 
+matplotlib
```

Comment:
These three libraries should go in requirements.txt rather than test-requirements.txt